docs/nidigital/class.rst

```diff
@@ -1188,72 +1188,13 @@ fetch_history_ram_cycle_information
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: fetch_history_ram_cycle_information(site, sample_index)
+    .. py:method:: fetch_history_ram_cycle_information(site, pin_list, position, samples_to_read)
```

Comment:
with sites becoming a repeated capability ( #1290 ) we will need to integrate this carefully.

Reply:
Yes, whichever PR goes last will merge the other's changes.